gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -694,6 +781,36 @@ gst_tensor_trainer_query_caps (GstTensorTrainer * trainer,
   return caps;
 }
 
+/**
+ * @brief Wait for training completion
+ */
+static gboolean
+gst_tensor_trainer_wait_for_training_completion (GstTensorTrainer * trainer)
+{
+  g_return_val_if_fail (trainer != NULL, FALSE);
+  g_return_val_if_fail (trainer->fw != NULL, FALSE);
```

Comment:
Is this line necessary?

Reply:
Yes, it check the sub-plugin (fw).

Reply:
Okay, I wil remove it

Reply:
I cannot see any lines that use ```fw``` in this function, can you?